app/models/dimensions/organisation.rb

```diff
@@ -0,0 +1,8 @@
+class Dimensions::Organisation < ApplicationRecord
+  validates :title, presence: true
+  validates :slug, presence: true
+  validates :link, presence: true
+  validates :content_id, presence: true
+  validates :organisation_state, presence: true
```

Comment:
I think we should drop the `organisation_` prefix here. It is duplication and should be handled during our ETL process where we translate between the external domain representation and our representation here.